package.json

```diff
@@ -25,17 +25,21 @@
   "type": "module",
   "main": "./index.mjs",
   "module": "./index.mjs",
+  "types": "types",
```

Comment:
does this automatically pick up on `types/index.d.ts`?

Reply:
yep, this is straight from the [dtslint docs](https://github.com/microsoft/dtslint#add-types-for-a-library-not-on-definitelytyped) and also how it's done [here](https://github.com/syntax-tree/xast-util-to-xml/blob/6491f5fb6ac0de84c774503a80192341bc7adb88/package.json#L29).